pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -285,8 +299,19 @@ func (config *V2RegistriesConf) postProcessRegistries() error {
 
 	for i := range config.Registries {
 		reg := &config.Registries[i]
-		// make sure Location and Prefix are valid
+		// make sure SubDomainPrefix, Location and Prefix are valid
 		var err error
+
+		// Prefer SubDomainPrefix over Location and Prefix
```

Comment:
Why do we prefer it over location?  Maybe I'll understand it when reading the rest of the PR but a more verbose comment would be good.

Reply:
Hmm, if subdomainprefix should conflict with prefix and location, then I will need to change this.

Reply:
addressed in upcoming commit.